controllers/kustomization_decryptor.go

```diff
@@ -160,6 +160,11 @@ func (kd *KustomizeDecryptor) ImportKeys(ctx context.Context) error {
 				if err := kd.gpgImport(keyPath); err != nil {
 					return err
 				}
+			case ".json":
+				keyPath := filepath.Join(kd.homeDir, name)
+				if err := os.WriteFile(keyPath, file, os.ModePerm); err != nil {
```

Comment:
We are writing all `.json` files but only seem to be interested in `azure_kv.json`, this has to change so it becomes consistent.

Reply:
that's right, but i have done it that way for two reasons :
- other file (.asc .agekey) works the same way, based on extension
- if someone add gcp support later, it probably also needs a json credential file

Reply:
> other file (.asc .agekey) works the same way, based on extension

But in that case the files are also taken into account.

> if someone add gcp support later, it probably also needs a json credential file

This would be another single JSON file, likely with a specific filename. Given this, I think something like the following would be better:

```
case keyservice.SOPSCredentialsFileAzureKeyvault:
	keyPath := filepath.Join(kd.homeDir, name)
	if err := os.WriteFile(keyPath, file, os.ModePerm); err != nil {
```